src/views/PDFView.vue

```diff
@@ -3,7 +3,13 @@
   - SPDX-License-Identifier: AGPL-3.0-or-later
   -->
 <template>
-	<iframe ref="iframe"
+	<div v-if="!isDownloadable"
+		id="emptycontent">
+		<div class="icon-error" />
+		<h2>{{ t('files_pdfviewer', 'To view a shared PDF file the sharer needs to allow downloads of the file') }}</h2>
```

Comment:
`To view a shared PDF file, the owner must allow a download of the file` ?

There is also a case, when files_pdfviewer is not in the whitelist of Guests app, and guest account is trying to access it with 403. 
Possibly unrelated, but I could follow-up on top of that PR later

Reply:
> `To view a shared PDF file, the owner must allow a download of the file` ?

Mmm, maybe the person should not be mentioned at all, because if a file is reshared and the owner disallows the download I guess (I have not checked) that it will be disallowed in the reshare, but if the owner allows it and the resharer disallows it then it would be also disallowed :thinking: 
 
> There is also a case, when files_pdfviewer is not in the whitelist of Guests app, and guest account is trying to access it with 403. Possibly unrelated, but I could follow-up on top of that PR later

Thanks, I was not aware of that. A follow up sounds good :-)

Reply:
any news on the follow up?

Reply:
This PR covers all shares (with users, guests, oublic links), where download is disabled.

For guests accounts + shareswith enabled download, it returns 500 Server error, which is correct, and not to be handled by PDF Viewer.
Instead, this app should be whitelisted in admin settings. Issue is tracked here: https://github.com/nextcloud/guests/issues/1249